### ISSUE TITLE
[5.x] y_va: Bare `___` usage found

### ISSUE THREAD

**Issue by Thomas14** [pawn-lang/YSI-Includes#370]
Hi,

Consider the following example code:

```Pawn
stock foo( arg1, arg2, arg3, va_args<> )
{
	// Some code to generate 'format'
	
	CallLocalFunction( "Bar", format, arg1, va_start<3> );
}
```

Results in:

> YSI Warning: Bare `___` usage found - make sure it is a function parameter.

I've tried looking at `y_va`'s code to get a clue about what's going on, but frankly I am not sure whether I am looking at a bug or simply using it wrong. I assume the latter as I would expect someone to have reported this already otherwise.

Any ideas?

**Comment by Y-Less** [comment]
Are you on the latest amx_assembly? I swear I've fixed this!

**Comment by Thomas14** [comment]
Whoops, can confirm it works on the latest amx_assembly!

Because of this I also just noticed I am quite silly. I promised to keep the submodules updated, so I occasionally clicked on the links in the repo, thinking "no new commits, still good!", not realizing that it takes you to the actual commit and so newer commits aren't immediately visible. Derp... well anyway, I've opened a PR for it #371 (and subscribed to the commit feed, so I won't miss updates again) :)

Sorry for the trouble ;)
